lib/core/sum.ex

```diff
@@ -25,7 +25,7 @@ defmodule TelemetryMetricsPrometheus.Core.Sum do
            :telemetry.attach(
              handler_id,
              metric.event_name,
-             &handle_event/4,
+             &__MODULE__.handle_event/4,
```

Comment:
Is there a reason you were changing these? Adding the module is not necessary.

Reply:
I didn't think it was necessary either, but someone pointed me to the telemetry docs a little while back https://hexdocs.pm/telemetry/ where it says to avoid local function captures. I don't know the inner BEAM workings on this one so just going off the docs.

```
Note: due to how anonymous functions are implemented in the Erlang VM, it is best to use function captures (i.e. fun mod:fun/4 in Erlang or &Mod.fun/4 in Elixir) as event handlers to achieve maximum performance. In other words, avoid using literal anonymous functions (fun(...) -> ... end or fn ... -> ... end) or local function captures (fun handle_event/4 or &handle_event/4 ) as event handlers.
```

Reply:
TIL that isn't limited to anonymous funs!